R/multi-req.R

```diff
@@ -44,19 +44,19 @@
 #'   request_base %>% req_url_path("/delay/0.5")
 #' )
 #' # But it's much faster if you request in parallel
-#' system.time(resps <- multi_req_perform(reqs))
+#' system.time(resps <- req_perform_multi(reqs))
 #'
 #' reqs <- list(
 #'   request_base %>% req_url_path("/status/200"),
 #'   request_base %>% req_url_path("/status/400"),
 #'   request("FAILURE")
 #' )
-#' # multi_req_perform() will always succeed
-#' resps <- multi_req_perform(reqs)
+#' # req_perform_multi() will always succeed
+#' resps <- req_perform_multi(reqs)
 #' # you'll need to inspect the results to figure out which requests fails
 #' fail <- vapply(resps, inherits, "error", FUN.VALUE = logical(1))
 #' resps[fail]
-multi_req_perform <- function(reqs, paths = NULL, pool = NULL, cancel_on_error = FALSE) {
+req_perform_multi <- function(reqs, paths = NULL, pool = NULL, cancel_on_error = FALSE) {
```

Comment:
Maybe this should be `req_perform_parallel()`? At least this sounds a bit more intuitive to me.

Reply:
Oh yeah, I like that name.